doc/installing.rst

```diff
@@ -67,12 +67,7 @@ server, and when it is successful you'll see two buttons: one to connect to
 your ownCloud Web GUI, and one to open your local folder. It will also start 
 synchronizing your files.
 
-.. image:: images/client-4.png
-   :alt: A successful server connection, showing a button to connect to your 
-    Web GUI, and one to open your local ownCloud folder
-
-Click the Finish button, and you're all done. 
-
 .. Links
    
 .. _ownCloud download page: https://owncloud.com/download/#desktop-clients
+.. _LTS: https://wiki.ubuntu.com/LTS
```

Comment:
@michaelstingl check if this was the link you wanted to ref. in https://github.com/owncloud/client/commit/d832312acc482a267207ffc89920786fd2a4b506#diff-6f76e4f0e29a50df7788fbc9f1a78901

Reply:
👍